PYTHON/index.json

```diff
@@ -84,5 +84,23 @@
             "data":"DataManipulation/transform_data_with_numpy.py"
          }
       ]
+   },
+   {
+      "name":"Data Types",
```

Comment:
Data Types is generic and may not convey the right message. Im thinking "UDF invocation details" or "Explore UDF data types" or "Invocation properties"